src/content.ts

```diff
@@ -219,13 +220,16 @@ config.getAsync("modeindicator").then(mode => {
         } else {
             result = mode
         }
-	if ( suffix != "" ) {
+	let showcmd = Config.get("showcmd")
```

Comment:
I don't like this name. I haven't thought of a better one yet. Perhaps `indicatorkeys`, and migrate `modeindicator` to `indicatormode`?

Posting this here so that I don't forget about it. I'm not expecting you to fix it : )